.github/workflows/win_exe_sign_test.yml

```
@@ -0,0 +1,236 @@
+      - name: Upload Espressif-IDE-macosx-cocoa-x86_64.dmg
+        if: ${{ !cancelled() }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: espressif-ide-macosx-cocoa-x86_64
+          path: releng/ide-dmg-builder/Espressif-IDE-macosx-cocoa-x86_64.dmg
+      
+      - name: Upload Espressif-IDE-macosx-cocoa-aarch64.dmg
+        if: ${{ !cancelled() }}
+        uses: actions/upload-artifact@v4
+        with:
+           name: espressif-ide-macosx.cocoa.aarch64
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 11

(indentation)